modules/components/src/Aggs/DatesAgg.js

```diff
@@ -1,145 +1,47 @@
 import React from 'react';
 import Moment from 'moment';
-import { DayPickerRangeController } from 'react-dates';
+import { DateRangePicker } from 'react-dates';
 import 'react-dates/initialize';
 import 'react-dates/lib/css/_datepicker.css';
-import { maxBy, minBy, sumBy } from 'lodash';
-import { css } from 'emotion';
-import {
-  inCurrentSQON,
-  replaceSQON,
-  toggleSQON,
-  removeSQON,
-} from '../SQONView/utils';
+import { maxBy, minBy } from 'lodash';
+import { replaceSQON } from '../SQONView/utils';
 import './AggregationCard.css';
 import AggsWrapper from './AggsWrapper';
 import './DatesAgg.css';
 
-const START_DATE_INPUT = 'startDate';
-const END_DATE_INPUT = 'endDate';
-const BUCKET_DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss.SSSSSS';
-const INPUT_DATE_FORMAT = 'YY/MM/DD';
-
+const BUCKET_DATE_FORMAT = 'YYYY-MM-DD';
 const bucketDateToMoment = dateString => Moment(dateString, BUCKET_DATE_FORMAT);
 const momentToBucketDate = moment => moment?.format(BUCKET_DATE_FORMAT);
-const inputDateToMoment = dateString => Moment(dateString, INPUT_DATE_FORMAT);
-const momentToInputDate = moment => moment?.format(INPUT_DATE_FORMAT);
-
-const DATE_PICKER_POSITIONS = {
-  BOTTOM_LEFT: css`
-    position: absolute;
-    left: 0%;
-    top: 100%;
-  `,
-  BOTTOM_RIGHT: css`
-    position: absolute;
-    right: 0%;
-    top: 100%;
-  `,
-  TOP_LEFT: css`
-    position: absolute;
-    left: 0%;
-    bottom: 100%;
-  `,
-  TOP_RIGHT: css`
-    position: absolute;
-    right: 0%;
-    bottom: 100%;
-  `,
-};
-
-const Arrow = ({ style }) => (
-  <svg
-    style={style}
-    className="DayPickerNavigation_svg__horizontal DayPickerNavigation_svg__horizontal_1"
-    viewBox="0 0 1000 1000"
-  >
-    <path d="M694.4 242.4l249.1 249.1c11 11 11 21 0 32L694.4 772.7c-5 5-10 7-16 7s-11-2-16-7c-11-11-11-21 0-32l210.1-210.1H67.1c-13 0-23-10-23-23s10-23 23-23h805.4L662.4 274.5c-21-21.1 11-53.1 32-32.1z" />
-  </svg>
-);
 
 class DatesAgg extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      localRange: this.getInitialRange(props), // the moment objects passed into DayPickerRangeController
-      inputRangeValues: {
-        // the strings to render in the inputs
-        startDate: null,
-        endDate: null,
-      },
+      ...this.getInitialRange(props),
       focusedInput: null, // must be set with this.setInputFocus to bind with dom state
```

Comment:
`this.setInputFocus` is not a thing anymore so we probably don't need the comment. In fact, do we still need this state? I think `DateRangePicker` can manage its own input state, unless we're controlling it with props :P

Reply:
@jgnieuwhof confirmed from doc that DateRangePicker needs the state managed outside: https://github.com/airbnb/react-dates#daterangepicker 

so... 🥇